### ISSUE TITLE
ARM architecture

### ISSUE THREAD

**Issue by christianbundy** [ssbc/patchwork#745]
I was recently advocating for a friend to install Patchwork on their Raspberry Pi, but it looks like the AppImage is the only supported release (and it doesn't look like they'll support ARM: https://github.com/AppImage/AppImageKit/issues/111).

I'm currently working through the manual installation, but it's a bit hairy for first-time users:

- Git clone
- Add Node.js apt repo
- Install Node.js
- Keyscan github.com
- Create swapfile
- NPM install

Are there any plans to provide non-x86 releases for Raspberry Pi etc?

**Comment by gmarcos87** [comment]
I use ssb on an rsp3, like pub. CPU consumption is incredibly high and I must use cpulimit if I don't lose control of the device. Even pulls down the wifi connection.
Anyway I think it's possible, try:

- download the patchwork source code
- npm install
- npm run rebuild
- npm run start

**Comment by christianbundy** [comment]
I'm using the same Raspberry Pi 3 (with Raspbian), but I had a few problems:

- The default Node.js version (4.9.2 I think?) is an unsupported sbot target, so I had to upgrade via the Node.js apt repository.
- `npm install` seems to pull via SSH, so I had to `ssh-keyscan` first.
- `npm install` runs out of memory, so I had to create a swapfile.
- `npm start` just shows that it's running `electron index.js` with no output.
